Installation/include/CGAL/config.h

```diff
@@ -351,6 +355,9 @@ using std::max;
 #endif
 
 // Macro CGAL_ASSUME and CGAL_UNREACHABLE
+#ifdef CGAL_CXX23
+#  define CGAL_ASSUME(EX) [[ assume(EX) ]]
+#else
```

Comment:
Please use [`elif`](https://en.cppreference.com/w/cpp/preprocessor/conditional) to avoid the nesting of conditionals. I find it more readable that way.

- If C++23,
- else if `__builtin_unreachable` exists,
- else if `__assume` exists...